packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -442,43 +477,69 @@ export default function ProjectNav({
         {!isGovernment && !isCloudManager && (
           <>
             <li role="none" className={productTabStyle}>
-              <a
-                data-testid="project-nav-realm"
-                href={projectNav.realm}
-                className={getProductClassName('realm')}
-                aria-disabled={!current}
-                tabIndex={current ? 0 : -1}
-                onClick={onElementClick(ProjectNavRealm)}
+              <Tooltip
+                {...sharedTooltipProps}
+                tabIndex={-1}
+                align="bottom"
+                justify="middle"
+                usePortal={false}
+                darkMode={true}
+                enabled={useCNRegionsOnly}
+                trigger={
+                  <a
+                    data-testid="project-nav-realm"
+                    href={useCNRegionsOnly ? hosts.cloud : projectNav.realm}
+                    className={getProductClassName('realm')}
+                    aria-disabled={!current}
+                    tabIndex={current ? 0 : -1}
+                    onClick={onElementClick(ProjectNavRealm)}
+                  >
+                    {!isMobile && (
+                      <RealmIcon
+                        {...productIconProp.prop}
+                        active={activeProduct === Product.Realm && isLoading}
+                        className={getIconStyle(Product.Realm)}
+                      />
+                    )}
+                    Realm
+                  </a>
+                }
               >
-                {!isMobile && (
-                  <RealmIcon
-                    {...productIconProp.prop}
-                    active={activeProduct === Product.Realm && isLoading}
-                    className={iconStyle}
-                  />
-                )}
-                Realm
-              </a>
+                {useCNRegionsOnlyMessage}
+              </Tooltip>
             </li>
 
             <li role="none" className={productTabStyle}>
-              <a
-                data-testid="project-nav-charts"
-                href={projectNav.charts}
-                className={getProductClassName('charts')}
-                aria-disabled={!current}
-                tabIndex={current ? 0 : -1}
-                onClick={onElementClick(ProjectNavCharts)}
+              <Tooltip
```

Comment:
Can we pull out the duplicated tooltip logic into a component? e.g.
```tsx
  const WithCNRegionsOnlyTooltip = useCallback(
    ({ children }: { children: React.ReactElement }) => {
      return (
        <Tooltip
          {...sharedTooltipProps}
          tabIndex={-1}
          align="bottom"
          justify="middle"
          usePortal={false}
          darkMode={true}
          enabled={useCNRegionsOnly}
          trigger={children}
        >
          {useCNRegionsOnlyMessage}
        </Tooltip>
      );
    },
    [sharedTooltipProps, useCNRegionsOnly],
  );

  ...

  <WithCNRegionsOnlyTooltip>
    <a
      data-testid="project-nav-charts"
      href={useCNRegionsOnly ? hosts.cloud : projectNav.charts}
      className={getProductClassName('charts')}
      aria-disabled={!current}
      tabIndex={current ? 0 : -1}
      onClick={onElementClick(ProjectNavCharts)}
    >
      {!isMobile && (
        <ChartsIcon
          {...productIconProp.prop}
          className={getIconStyle(Product.Charts)}
          active={activeProduct === Product.Charts && isLoading}
        />
      )}
      Charts
    </a>
  </WithCNRegionsOnlyTooltip>
```

Reply:
👍